recipes/erlang/build.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+export LIBRARY_PATH="${PREFIX}/lib:${LIBRARY_PATH}"
+export LD_LIBRARY_PATH="${PREFIX}/lib:${LD_LIBRARY_PATH}"
+export ERL_TOP="$(pwd)"
+./configure --with-ssl="${PREFIX}" --prefix="${PREFIX}" --without-javac
```

Comment:
Could we please add `"--enable-m${ARCH}-build"` too? This helps with 32-bit and 64-bit builds.